script.js

```diff
@@ -168,10 +168,16 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 $( document ).ready(function() {
+
   $('a.submit-a-request').each(function () {
-    if (location.pathname.indexOf('/requests/new') > -1) {
+    if (location.pathname.endsWith('/requests/new')) {
       $(this).hide();
       $('.contact-block').hide();
+
+      $("form.search").submit(function() {
+        $("form.search").attr('target', '_blank');
+        return true;
```

Comment:
what is the use case of returning `ture` here?

Reply:
typo. not necessary